PSScriptAnalyzerSettings.psd1

```diff
@@ -1,7 +0,0 @@
-@{
```

Comment:
Why was this deleted?

Reply:
This should've been removed during https://github.com/Lombiq/PowerShell-Analyzers/issues/12, so I added the explanation in the commit message.